main.cpp

```diff
@@ -0,0 +1,419 @@
+#include <SFML/Graphics.hpp>
+#include <SFML/Network.hpp>
+#include <iostream>
+#include <windows.h>
+#include <map>
+#include "TextureHolder.h"
+#include "Square.h"
+#include "Piece.h"
+
+bool move(Square* slctd, Square* trgt, sf::UdpSocket& sckt, sf::IpAddress& IP, unsigned short& PORT, sf::Packet& pck);
+
+int main()
+{	
+	//kod debugowania = 1337
```

Comment:
Po co taki komentarz, jeżeli można go zapisać kodem (i zawrzeć więcej informacji)
```cpp
constexpr std::uint16_t debugCode = 1337;
```
Wtedy mamy taki przyjemny zapis: `if (serverPort != debugCode)` i nie trzeba komentarza.
btw. jeżeli przyrównujemy do portu, to może debugPort?

Jeżeli była by to klasa to takie rzeczy można dać do pola private.